OpenMcdf.Ole/OpenMcdf.Ole.csproj

```diff
@@ -0,0 +1,17 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFrameworks>netstandard2.0;net8.0</TargetFrameworks>
+    <LangVersion>12.0</LangVersion>
+    <ImplicitUsings>enable</ImplicitUsings>
+    <Nullable>enable</Nullable>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="System.Collections.Immutable" Version="8.0.0" />
```

Comment:
I'll try to make time to do some tests later, but one small thing here - can these nuget references be conditionalized so they're only included for .NET Standard 2?
(refs comments in https://github.com/ironfede/openmcdf/pull/123 about them being built in to .NET)

Reply:
@Numpsy Good call, I'd missed that a package was available for it for netstandard2.0.